oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsDisks.java

```diff
@@ -70,6 +70,31 @@
 
     private static final int BUFSIZE = 255;
 
+    public static void updateDiskStats(HWDiskStore diskStore) {
+        Map<String,List<Object>> vals = WmiUtil.selectObjectsFrom(null, "Win32_DiskDrive",
```

Comment:
I'm not sure we need to read anything from WMI for this update.  None of these values change.

Reply:
That's a good point, I think my mind was somewhere else when I was doing the Windows bit.  I'll have another go in a bit.